doc/nrf/releases/release-notes-latest.rst

```diff
@@ -132,6 +132,16 @@ Matter (Project CHIP)
 
   * Renamed occurrences of Project CHIP to Matter.
 
+Thread
+------
+
+* Added support for the following Thread 1.2 features:
+
+  * Link Metrics
+  * Coordinated Sampled Listening (CSL) Receiver
```

Comment:
CSL Transmitter is also supported, but built by default when Thread Version 1.2 is selected. So I would remove the Receiver part from here.